### ISSUE TITLE
An internal error occurred during: "Loading section 'All Repositories'"

### ISSUE THREAD

**Issue by atmcq** [microsoft/team-explorer-everywhere#345]
Seeing the error below on trying to access Azure Devops repositories, following new install of TEE:

```
An internal error occurred during: "Loading section 'All Repositories'"
java.lang.NoSuchMethodError: 'org.eclipse.egit.core.RepositoryUtil org.eclipse.egit.core.Activator.getRepositoryUtil()'
	at com.microsoft.tfs.client.eclipse.ui.egit.teamexplorer.TeamExplorerGitRepositoriesSection.loadRegisteredRepositories(TeamExplorerGitRepositoriesSection.java:215)
	at com.microsoft.tfs.client.eclipse.ui.egit.teamexplorer.TeamExplorerGitRepositoriesSection.initialize(TeamExplorerGitRepositoriesSection.java:100)
	at com.microsoft.tfs.client.eclipse.ui.egit.teamexplorer.TeamExplorerGitRepositoriesSection.initialize(TeamExplorerGitRepositoriesSection.java:105)
	at com.microsoft.tfs.client.common.ui.controls.teamexplorer.TeamExplorerBaseControl$SectionLoadJob.run(TeamExplorerBaseControl.java:481)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
```

Eclipse 4.23
TEE 14.137.0

**Comment by howlger** [comment]
This is caused by [this](https://github.com/microsoft/team-explorer-everywhere/issues/353#issuecomment-1513901828).

As workaround downgrade JGit/EGit to 5.11 or use Eclipse 2021-03.

See also [my Stack Overflow answer here](https://stackoverflow.com/a/76049707/6505250).

**Comment by eric-milles** [comment]
I may be able to fix this.  `RepositoryUtil` is still there but is now an enum.  There are 4 spots in the code that make reference to `Activator#getRepositoryUtil`.  @howlger Can you provide steps to get to the error?  Do I need a git-based ADO server?

**Comment by howlger** [comment]
> @howlger Can you provide steps to get to the error? Do I need a git-based ADO server?

Sorry, but I'm not an user of the TFS plugin. I saw this [Stack Overflow issue with the stack trace](https://stackoverflow.com/q/76045184/6505250) and I knew about the change in EGit that caused it, so I looked into the code and shared my findings. Maybe one of the issue reporters, @atmcq and @kris-mart, or the one who asked on Stack Overflow can tell how to reproduce it.
